lib/core/src/clientLogin.cpp

```diff
@@ -479,7 +504,7 @@ clientLoginWithPassword( rcComm_t *Conn, char* password ) {
     memset( md5Buf, 0, sizeof( md5Buf ) );
     strncpy( md5Buf, authReqOut->challenge, CHALLENGE_LEN );
     setSessionSignatureClientside( md5Buf );
-
+    set_session_signature_client_side(Conn, md5Buf);
```

Comment:
Perhaps the documentation around this function would be enough, but maybe we should leave a comment to remind our future selves why we are calling both `setSessionSignatureClientside` and `set_session_signature_client_side`?

Reply:
Done